test/pinnedTabTest.js

```diff
@@ -0,0 +1,28 @@
+/* global describe, it, before */
+
+const Brave = require('./lib/brave')
+
+const messages = require('../js/constants/messages')
+const {urlInput} = require('./lib/selectors')
+
+describe('pinnedTabs', function () {
+  function * setup (client) {
+    yield client
+      .waitUntilWindowLoaded()
+      .waitForVisible('#window')
+      .waitForVisible(urlInput)
+  }
+
+  describe('new tab signal', function () {
+    Brave.beforeAll(this)
+    before(function *() {
+      yield setup(this.app.client)
+    })
+
+    it('creates a new pinned tab when signaled', function *() {
+      yield this.app.client
+        .ipcSend(messages.SHORTCUT_NEW_FRAME, 'http://www.brave.com', { isPinned: true })
+        .waitForExist('.tab[data-frame-key="2"]')
```

Comment:
Please update this to: `.tab.isPinned[data-frame-key="2"]` and make sure the test still works.